vortex-ffi/src/lib.rs

```diff
@@ -12,26 +12,13 @@ mod log;
 mod stream;
 
 use std::ffi::{CStr, c_char, c_int};
-use std::sync::LazyLock;
 
 pub use log::vx_log_level;
-use tokio::runtime::{Builder, Runtime};
-use vortex::error::VortexExpect;
 
 #[cfg(all(feature = "mimalloc", not(miri)))]
 #[global_allocator]
 static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
 
```

Comment:
~~Double check whether we can remove tokio from the `Cargo.toml` in `vortex-ffi`.~~